addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/handler/RFXComBridgeHandler.java

```diff
@@ -153,6 +152,11 @@ public void initialize() {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
                     "Please use the Transceiver over TCP/IP bridge type for a serial over IP connection.");
             return;
+        } else if (configuration.serialPort == null && configuration.bridgeId != null) {
```

Comment:
Can serialPort be null as parameter is now required/mandatory in thing description?

Reply:
Yes, it will be just after installing the new binding for existing bridges that have bridgeId but not serialPort.
For later created bridges with Paper UI, it will no more be possible as the field is required.